common/persistence/visibility/store/elasticsearch/visibility_store.go

```diff
@@ -469,97 +460,25 @@ func (s *visibilityStore) ScanWorkflowExecutions(
 	ctx context.Context,
 	request *manager.ListWorkflowExecutionsRequestV2,
 ) (*store.InternalListWorkflowExecutionsResponse, error) {
-	// Elasticsearch 7.10+ can use "point in time" (PIT) instead of scroll to scan over all workflows without skipping or duplicating them.
-	// https://www.elastic.co/guide/en/elasticsearch/reference/7.10/point-in-time-api.html
-	if s.esClient.IsPointInTimeSupported(ctx) {
-		return s.scanWorkflowExecutionsWithPit(ctx, request)
```

Comment:
What happens to users who were running with ES 7.10+ and relied on this query returning no duplicates? If there are such users, I think we should talk to the community (sooner the better) about this change, so that they can prepare for it.

Reply:
The documentation around Scan API (https://docs.temporal.io/tctl-v1/workflow#scan) doesn't says anything about guaranteeing no duplicates. In fact, it compares to List API, but faster and not sorted.